.github/workflows/build-deploy-pr.yaml

```diff
@@ -37,4 +37,4 @@ jobs:
       - name: deploy pull request container
```

Comment:
you could delete all the deploy stuff since we won't be using that.  the image will still live on dockerhub and be released using "manual Make", but github actions will test build and run it

Reply:
Cool, got it. It looks like the test is erroring out when it tries to login to Docker. I've had troubles with that lately and have had to manually login to Docker when I do a "make release". For some reason it never finds my secrets file with my token